configure.ac

```diff
@@ -1,12 +1,13 @@
 AC_INIT([readstat], [20160511])
 AM_INIT_AUTOMAKE([foreign subdir-objects])
 AM_SILENT_RULES([yes])
-LT_INIT([disable-static])
+LT_INIT([disable-static, win32-dll])
```

Comment:
Same here, I'm not sure what that really does and whether this also makes sense on non-Windows platforms. Without it nothing compiled...